walkthroughs/howto-tls-file-provided/src/customEnvoyImage/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG AWS_ENVOY_IMAGE
+FROM ${AWS_ENVOY_IMAGE}
+
+RUN yum install -y jq
+RUN yum install -y curl
+RUN yum install -y unzip
+RUN curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+RUN unzip awscli-bundle.zip
+RUN ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
```

Comment:
Minor: You can combine these steps into the same `RUN` command, which will reduce the amount of layers and overall space used:

```
RUN yum install -y jq curl unzip && \
    curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip" && \
    unzip awscli-bundle.zip && \
    ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws && \
    rm -rf awscli-bundle.zip
```